File: BECore/app/model/ProjectClose_AttachDetail.js
/// <summary>
/// Model EXTJS ProjectClose_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectClose_AttachDetail', {
    extend: 'Ext.data.Model',

    idProperty: 'ProjectClose_AttachDetailID',
    idType: 'Guid',

    fields: [
        { name: "ProjectClose_AttachDetailID", type: "string" },
		{ name: "ProjectCloseID", type: "string" },
        { name: "Contents", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "IsTemp", type: "boolean" },
    ]
});

File: MT.Model/Business/ProjectClose_AttachDetail.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectClose_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectClose_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectClose_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectClose_AttachDetailID { get; set; }
        public Guid ProjectCloseID { get; set; }
        public string Contents { get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }

        public Guid? FileResourceID { get; set; }

    }
    #endregion
}

